src/compiler/checker.ts

```diff
@@ -46384,11 +46384,12 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
                     );
                 }
 
-                if (!isIllegalExportDefaultInCJS && getIsolatedModules(compilerOptions) && !(sym.flags & SymbolFlags.Value)) {
+                if (!isIllegalExportDefaultInCJS && !(node.flags & NodeFlags.Ambient) && getIsolatedModules(compilerOptions) && !(sym.flags & SymbolFlags.Value)) {
+                    const nonLocalMeanings = getSymbolFlags(sym, /*excludeTypeOnlyMeanings*/ false, /*excludeLocalMeanings*/ true);
                     if (
                         sym.flags & SymbolFlags.Alias
-                        && resolveAlias(sym) !== unknownSymbol

```

Comment:
`unknownSymbol.flags === SymbolFlags.All`, so while this check was necessary before, it’s now redundant with `!(nonLocalMeanings & SymbolFlags.Value)`.

Reply:
They're actually equal to `All`? I thought that it was `SymbolFlags.Property` for this symbol.

Reply:
Huh, oops, you’re right. At any rate, it’s still redundant with `!(nonLocalMeanings & SymbolFlags.Value)`, but I’m glad to fix that memory